content/kubernetes/deployment/quick-start.md

```diff
@@ -38,48 +33,52 @@ To deploy the Redis Enterprise operator, you'll need:
 * a Kubernetes client (kubectl)
 * access to DockerHub, RedHat Container Catalog, or a private repository that can hold the required images.
 
-See your version's [release notes]({{<relref "/kubernetes/release-notes/_index.md">}}) for a list of required images.
-
-If you are not pulling images from DockerHub, see [Private Repositories](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/README.md#private-repositories) for additional sections to add to your REC resource file and your `operator.yaml` file.
-
 ### Create a new namespace
 
-The Redis Enterprise operator manages a single Redis Enterprise cluster in a single namespace.
+**Important:** Each namespace can only contain one Redis Enterprise cluster. Multiple RECs with different operator versions can co-exist on the same Kubernetes cluster, as long as they are in separate namespaces.
 
-Throughout this guide, you should assume that each command is applied to the namespace in which the Redis Enterprise cluster operates.
+Throughout this guide, each command is applied to the namespace in which the Redis Enterprise cluster operates.
 
 1. Create a new namespace
 
     ```sh
-    kubectl create namespace <my-namespace>
+    kubectl create namespace <rec-namespace>
     ```
 
 2. Change the namespace context to make the newly created namespace default for future commands.
 
     ```sh
-    kubectl config set-context --current --namespace=<my-namespace>
+    kubectl config set-context --current --namespace=<rec-namespace>
     ```
 
+You can use an existing namespace as long as it does not contain any existing Redis Enterprise cluster resources. It's best practice to create a new namespace to make sure there are no Redis Enterprise resources that could interfere with the deployment.
+
+
 ## Install the operator
 
-The Redis Enterprise operator implementation is published as a Docker container. For more information about image sources, see [Manage container images]({{<relref "/kubernetes/deployment/container-images.md">}})
+The Redis Enterprise operator bundle is published as a Docker container. A list of required images is available in the [release notes]({{<relref "/kubernetes/release-notes/_index.md">}}) for each version.
```

Comment:
Container technology has long been standardized beyond Docker which is now just another company in the container space. The correct term to use is **container image**.

```suggestion
The Redis Enterprise operator bundle is published as a container image. A list of required images is available in the [release notes]({{<relref "/kubernetes/release-notes/_index.md">}}) for each version.
```